backend/data/processors/manualFixes/COMMFixes.py

```diff
@@ -0,0 +1,333 @@
+        "processed": "COMM1140. Only available to single && double degree Business School in Term 2. It will be offered to non-Business School in Term 3"
+    """
+
+    return {
+        "original": conditions["original"],
+        "processed": "COMM1140",
+        "warning": "Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3."
+    }
+
+# def COMM_1190(conditions):
+    """
+        "original": "Pre-requisite: COMM1110 or ECON1203 or MATH1041 or MATH1151 or MATH1131 or MATH1141. Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3.<br/><br/>",
+        "processed": "COMM1110 || ECON1203 || MATH1041 || MATH1151 || MATH1131 || MATH1141. Only available to single && double degree Business School in Term 2. It will be offered to non-Business School in Term 3"
+    """
+    
+    return {
+        "original": conditions["original"],
+        "processed": "COMM1110 || ECON1203 || MATH1041 || MATH1151 || MATH1131 || MATH1141",
+        "warning": "Only available to single and double degree Business School students in Term 2. It will be offered to non-Business School students in Term 3."
+
+    }
+
+# def COMM_1900():
+    """
+        "original": "Enrolment is only permitted for students in program Actl/Comm (3155) or Comm/Econ (3521).<br/>Pre-req: (ECON1101 and ECON1102) or (DPBS1101 and DPBS1102)<br/><br/>",
+        "processed": "Enrolment is only permitted for in program Actl/Comm (3155) || Comm/Econ (3521). (ECON1101 && ECON1102) || (DPBS1101 && DPBS1102)"
+    """
+
+    return "(3155 || 3521) && ((ECON1101 && ECON1102) || (DPBS1101 && DPBS1102))"
+
+def COMM_1999(conditions):
+    """
+        "original": "Pre-requisite: COMM0999 AND completed 30 UOC of Integrated First Year Core<br/><br/>",
+        "processed": "COMM0999 && 30UOC of Integrated First Year Core"
+    """
+
+    return {
+        "original": conditions["original"],
+        "processed": "COMM0999",
+        "warning": "You must have completed 30UOC of Integrated First Year Core"
+    }
+
+def COMM_2222(conditions):
+    """
+        "original": "Pre-requisite: In Business degree, COMM6000 Career Accelerator or COMM1999 First Year Portfolio, WAM 65+, minimum 72 UOC completed, room in degree for course, Good academic standing.<br/>Excluded: Co-op students<br/>It is recommended a progression check is completed prior to enrolling.<br/><br/>",
+        "processed": "In Business degree || COMM6000 Career Accelerator || COMM1999 First Year Portfolio, 65WAM, 72UOC , room in degree for course, Good academic standing"
+    """
+
+    return  {
+        "original": conditions["original"],
+        "processed": "(COMM6000 || COMM1999) && ZBUS && 65WAM && 72UOC"
+    }
```

Comment:
Change ZBUS to ZBUS#

Add in warning field about good standing and how co-op students are excluded